lib/rack/tracker/google_global/google_global.rb

```diff
@@ -12,10 +12,10 @@ def params
   class Event < OpenStruct
     PREFIXED_PARAMS = %i[category label]
     LITERAL_PARAMS  = %i[value]
```

Comment:
is this obsolete then? 🤔

Reply:
Yes, you are right - `LITERAL_PARAMS` are not used anywhere.
Should I fix that or you will?

Reply:
yeah please fix that :)

Reply:
Removed that constant.